src/components/RewardWrapper/index.tsx

```diff
@@ -149,15 +157,11 @@ export function RewardWrapper({
   // Translation override
   const t = tOverride || tOriginal
 
-  let isEligible = false
-
-  if (questPlayStreakData) {
-    const playstreakQuestStatus = getPlaystreakQuestStatus(
-      questMeta,
-      questPlayStreakData
-    )
-    isEligible = playstreakQuestStatus === 'READY_FOR_CLAIM'
-  }
+  const isEligible = canClaimReward({
+    quest: questMeta,
+    externalEligibility: externalEligibility,
+    playstreakData: questPlayStreakData
+  })
```

Comment:
we need both this and `useCanClaimReward` in this component?

Reply:
We will show the button but disabled